1_fetch.R

```diff
@@ -46,9 +47,39 @@ p1_targets_list <- list(
   tar_target(
     p1_nhd_reaches,
     download_nhdplus_flowlines(p1_drb_comids_all_tribs$COMID)
-  )
-
+  ),
+  
+  # STATSGO SOIL Characteristics
+  ## get selected child items nhdv2 STATSGO Soil Characteristics
+  ## 1) Text attributes and 2) Layer attributes
+  tar_target(
+    p1_selected_statsgo_sbid_children,
+    item_list_children(sb_id = nhd_statsgo_parent_sbid) %>% 
+      Filter(function(x){str_detect(x[['title']],'Text|Layer')},
```

Comment:
Huh, that's neat. I'm not familiar with the `Filter()` function. Is this in `purrr`?

Reply:
Actually, I believe `Filter()` is base. can filter items of a vector. Sort of the same family as `Reduce()`.